xikolo-ios/Layouts/de.lproj/TabSettings.strings

```diff
@@ -1,18 +1,30 @@
 
-/* Class = "UIBarButtonItem"; title = "Login"; ObjectID = "Mnf-wa-pbl"; */
-"Mnf-wa-pbl.title" = "Einloggen";
+/* Class = "UILabel"; text = "[build]"; ObjectID = "BYP-h5-ZvT"; */
+"BYP-h5-ZvT.text" = "[build]";
```

Comment:
I'd delete these "translations", it's easier to find what you're looking for in larger storyboards.